packages/passport-interface/src/api/requestCheckInById.ts

```diff
@@ -0,0 +1,75 @@
+import { ArgumentTypeName } from "@pcd/pcd-types";
+import { SemaphoreIdentityPCDPackage } from "@pcd/semaphore-identity-pcd";
+import { SemaphoreSignaturePCDPackage } from "@pcd/semaphore-signature-pcd";
+import { Identity } from "@semaphore-protocol/identity";
+import urlJoin from "url-join";
+import {
+  CheckTicketInByIdError,
+  CheckTicketInByIdRequest,
+  CheckTicketInByIdResponseValue,
+  ISSUANCE_STRING
+} from "../RequestTypes";
+import { APIResult } from "./apiResult";
+import { httpPost } from "./makeRequest";
+
+/**
+ * Tries to check the user in. This is called by the Zupass client when
```

Comment:
mention that this is for devconnect tickets only

Reply:
It says this on the next line